tests/unittest/benchmark_runner/common/workload_flavors/golden_files/func_ci_vdbench_pod_OCS_PVC_False/vdbench_pod.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
```

Comment:
vdbench must run on OCS, no meaning to the test OCS_PVC_false

Reply:
Yes; this is just the automated golden file checker, and I don't want to have special cases inside it.  This file is unneeded in practice but harmless.